### DISCUSSION TITLE
Build failure because reading https://go.tmz.dev/musttag?go-get=1: 404 Not Found

### DISCUSSION THREAD

**Discussion by akitanaka** [golangci/golangci-lint#4343] [Q&A] [answered]
We suddenly started seeing build failure because reading https://go.tmz.dev/musttag?go-get=1: 404 Not Found.

```
../../../../pkg/mod/github.com/golangci/golangci-lint@v1.54.2/pkg/golinters/musttag.go:4:2: unrecognized import path "go.tmz.dev/musttag": reading https://go.tmz.dev/musttag?go-get=1: 404 Not Found
make: *** [Makefile:210: install-tools] Error 1
```

I think we can avoid the build failure with this commit (https://github.com/golangci/golangci-lint/commit/89bfdc0503b923358615576dc14a4a42faf25f46) but the commit isn't in the latest release. Could you please consider releasing a new release (or fix the build issue)?

**Comment by ldez** [comment]
Hello,

Maybe @tmzane can restore this path temporarily?

@akitanaka you can use our binaries or our Docker image https://golangci-lint.run/usage/install/#local-installation
And if you are using the "tools pattern", you can use a more recent commit.

Note: the "tools pattern" is not recommended https://golangci-lint.run/usage/install/#install-from-source

**Comment by tmzane** [comment, marked as answer]
@akitanaka, I restored the old path, could you try again?

For some reason, I thought there was already a new version of golangci-lint with the new path, so I disabled the old one.

Please note that I'll have to disable it in about a month anyway due to domain ownership issues (the reason I migrated to the new path in the first place). Sorry for the inconvenience.

@ldez, thanks for the ping.

**Comment by Issacwww** [comment]
Hey @tmzane , run into the similar issue
```
go get go.tmz.dev/musttag                 
go: unrecognized import path "go.tmz.dev/musttag": https fetch: Get "https://go.tmz.dev/musttag?go-get=1": dial tcp: lookup go.tmz.dev on 127.0.0.1:53: no such host
``` 
is there a new host to consume?

**Reply by ldez** [reply]
You are using an old version of golangci-lint: currently golangci-lint doesn't depend on `go.tmz.dev/musttag` since 1 year.
The module name is now `go-simpler.org/musttag` and golangci-lint uses it.

So you must update your version of golangci-lint.

Also, installations from sources (`go build`/`go install`/etc.) are not recommended.
https://golangci-lint.run/welcome/install/#install-from-sources

Please update golangci-lint and use the official binaries.
